libnavigation-core/src/test/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSessionTest.kt

```diff
@@ -107,72 +99,6 @@ class MapboxDirectionsSessionTest {
         verify { router.cancel() }
     }
 
-    @Test
-    fun fasterRoute_timerStartedOnce() {
```

Comment:
Can we recreate these test in `MapboxNavigation`?

Reply:
I am working on it

Reply:
Still valid.